source/develop/release-management/features/network/ipv6_subnets.html.md

```diff
@@ -152,21 +381,165 @@ provision the following networking API entities, represented in yaml:
+| dynamic IPs             |      supported, if prefix == 64     |     supported    |
+| allowed prefix lengths  |            64 <= X <= 128           |      64 only     |
+| ipv6 privacy extensions |            not supported            |     supported    |
+| isolated subnet         | if guest routes manually configured |   not supported  |
+
+A brief explanation of each of the use cases described in the tables can be
+found below.
+
+#### Fixed IPs
+
+Fixed IPs are not supported on dhcpv6-stateless since the client itself
+auto-generates the IP address based on the advertised network prefix.
+This means the IP address ***is not*** sent in the DHCPv6 Reply message, thus
+making the guest VM IP address configuration impossible.
+
+#### Allowed prefix lengths and dynamic IPs
+
+The [slaac](https://tools.ietf.org/html/rfc4862) RFC indicates that all network
+prefixes **are** 64 bit long. Since on DHCPv6 stateless the IP prefix
+configuration is done as in slaac, the IPv6 subnets having dhcpv6_stateless
+address mode are thus limited to /64 prefixes.
+
+On DHCPv6_stateful, the IP address is sent on the DHCPV6 Reply message, and the
+network prefixes - and lengths - are communicated through RA messages.
+
+Thus, the guest is able to configure its addresses - and routes - to communicate
+with subnets having longer prefixes.
+
+There is a caveat though: dynamic IP addresses cannot be used when the network
+prefix length is different than 64,  meaning they will have to be configured
+through the networking API [fixed IPs](https://developer.openstack.org/api-ref/network/v2/?expanded=create-subnet-detail,create-port-detail#create-port)
+create port method - specifying both the subnet id, and the IP address.
+Since fixed IP addresses are required, this feature would only be possible if
+the logical switch ports are updated through the RESTful API.
+
+This implies that dynamic IP addresses are **only possible** on dhcpv6_stateful
+address mode **if** the network prefix length is 64 bit long.
+
+#### IPv6 privacy extensions
+
+Privacy extensions are defined in [rfc](https://tools.ietf.org/html/rfc4941).
+They can **only** be applied to stateless address modes, and are a matter
+of guest VM configuration.
+
+NetworkManager could be used to configure the privacy extensions per interface:
+```bash
+nmcli conn modify <interface_name> ipv6.ip6-privacy 2
+```
+
+This setting will be applied automatically, and the new temporary IP address
+will be used as the default.
+
+A caveat about privacy extensions is that they **render impossible** IP spoofing
+protection if used, since OVN central will not be able to know the port's IP
+address during provisioning time.
+
+#### Isolated subnet
+
+Isolated subnets - not attached to OVN logical routers - can be achieved if the
+address mode is set to dhcpv6_stateful. The caveat here is that the user/admin
+has to configure the guest VM routes manually, since there is no router to send
```

Comment:
The guest VM network prefix length has to configured manually. Why should be routes required, if there is no router?

Reply:
You're right, I was thinking of the scenarios where I had interconnected subnets, which, by definition, are not isolated. My bad.

Got rid of that sentence.